### PR TITLE
Ed sheeran music

### PR DESCRIPTION
**Pull request by BlackBeaner**
Please review me ,@casey-collab

### CODE REVIEW

File: english/Ed-Sheeran-music/list-song.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+List of songs
+===============================
+1.[Happier-Ed Sheeran](Happier/happier-lyric.md)<br/>
+2.[Perfect-Ed Sheeran](Perfect/perfect-lyric.md)
```
Comment (casey-collab) [inline english/Ed-Sheeran-music/list-song.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by BlackBeaner** [comment]
Please review again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**